### PR TITLE
IL-884 Make all non-deprecated tracks use Instruqt Version 3 configuration

### PR DESCRIPTION
**Pull request by thomashashi**
Pre-req to https://github.com/hashicorp/field-workshops-consul/pull/236

### CODE REVIEW

File: instruqt-tracks/consul-basics/track.yml [hunk 1]
```diff
@@ -27,350 +26,10 @@ tags:
...
-    type: code
-    hostname: consul-server-2
-    path: /tmp/bootstrap.txt
-  difficulty: basic
-  timelimit: 600
+lab_config:
+  overlay: false
+  width: 0
+  position: ""
 checksum: "4620626064868703026"
```
Comment (omkensey) [inline instruqt-tracks/consul-basics/track.yml:35]: The checksums probably need to be stripped out.

File: instruqt-tracks/consul-basics/01-meet-consul/assignment.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+slug: 01-meet-consul
+id: bgfx8nbzqnjv
```
Comment (omkensey) [inline instruqt-tracks/consul-basics/01-meet-consul/assignment.md:3]: Are challenge IDs still valid after the move?

File: instruqt-tracks/service-discovery-with-consul/track.yml [hunk 3]
```diff
@@ -19,284 +20,10 @@ tags:
...
-  - title: App Server
-    type: terminal
-    hostname: app
-  difficulty: basic
-  timelimit: 900
+lab_config:
+  overlay: false
+  width: 0
+  position: ""
 checksum: "12769099385139066651"
```
Comment (omkensey) [inline instruqt-tracks/service-discovery-with-consul/track.yml:29]: Same comment here about checksums.

**Review by omkensey — COMMENTED** [review]
In general I'm assuming that the simple track format change causes no issues, but I noted a few things I'm not sure of.

**Comment by thomashashi** [comment]
> In general I'm assuming that the simple track format change causes no issues, but I noted a few things I'm not sure of.

It shouldn't cause any issues (if it does, Instruqt has failed in their conversion process). Good catches on the `id:` and `checksum:` fields, I'll remove those. Longer term I'd like to remove those from all the tracks, it keeps the noise down and makes things less fragile for us, I feel.

**Comment by thomashashi** [comment]
Just pushed a new bit to this branch, removing all `id:` and `checksum:` fields.

**Review by omkensey — APPROVED** [review]
LGTM

**Pull request merged**